e2e/po/south.page.ts

```diff
@@ -0,0 +1,67 @@
+import { browser, by, element } from 'protractor';
+
+export class SouthPage {
+  EC = browser.ExpectedConditions;
+
+  navToSouthPage() {
+    return browser.get('/#/south');
+  }
+
+  getSouthPageTitle() {
+    browser.ignoreSynchronization = true;
+    return element(by.css('#scheduled-process .title')).getText();
+  }
+
+  clickAddServiceButton() {
+    browser.ignoreSynchronization = true;
+    return element(by.id('add_south_service')).click();
+  }
+
+  addSouthService() {
+    browser.ignoreSynchronization = true;
+    browser.sleep(2000); // wait to get south service plugin from server
+    element(by.name('type')).all(by.tagName('option'))
+      .then(options => {
+        options[0].click();  // select first plugin in select box
+      });
+    element(by.id('name')).sendKeys('demo-service'); // supply service name
```

Comment:
instead of keeping this in page object, can we pass from test? I would like the name to be `guiE2eService #1`